src/assets/translations/en/main.json

```diff
@@ -1315,5 +1315,13 @@
         "neverSellData": "Never sell data for profit"
       }
     }
+  },
+  "features": {
+    "yat": {
+      "banner": {
+        "title": "Get a Yat",
+        "description": "Your Yat is your universal emoji username, website&nbsp;URL, payment address, and more."
```

Comment:
This doesn't properly encode when used as a translation

<img width="1226" alt="image" src="https://user-images.githubusercontent.com/17035424/185933601-7cb2abe8-8984-4f8a-9592-b36751ca8300.png">

What are we trying to achieve here? Looking at the mocks, we shouldn't need this. I get it why we would want "website URL" to always be displayed as one subject on all viewports, but it would mean splitting the translations which make globalization work more difficult. I'd simply remove it and use a regular space.